src/Elements/WrapperStop.php

```diff
@@ -130,24 +129,19 @@ protected function compile()
 			// check if the scripts should be combined
 			$combine = '';
 
-			// ptable for swiperStop is "tl_article"
-			// get the article-model for the content-model
-			$article = ArticleModel::findById((int)$objContent->pid);
-			if ($article) {
-				// get the page-model of the article-model
-				$page = PageModel::findById((int)$article->pid);
-
-				// if the page itself has no layout, check for a parent-page
-				while ($page && !$page->includeLayout) {
-					$page = PageModel::findById($page->pid);
-				}
-				if ($page && $page->layout) {
-					$layout = LayoutModel::findById((int)$page->layout);
-					// get the current layout-model of the page
-					if ($layout) {
-						if ($layout->add_swiper_scripts) {
-							$combine = '|static';
-						}
+			// get the current page
+			global $objPage;
+			$page = $objPage;
+			// if the page itself has no layout, check for a parent-page
+			while ($page && !$page->includeLayout) {
+				$page = PageModel::findById($page->pid);
+			}
```

Comment:
The loop is not necessary. The page object should already have that information loaded.

Reply:
removed the loop, so that it is always used the layout-value of `global $objPage`